src/backend/parser/cypher_expr.c

```diff
@@ -955,6 +956,10 @@ static Node *transform_cypher_typecast(cypher_parsestate *cpstate,
     {
         fname = lappend(fname, makeString(FUNC_AGTYPE_TYPECAST_PG_BIGINT));
     }
+    else if ((pg_strcasecmp(ctypecast->typecast, "bool") == 0 || pg_strcasecmp(ctypecast->typecast, "boolean") == 0))
```

Comment:
Please make sure to conform to our ~80 character per line coding standard.

Reply:
Done